src/loaders/compute_tools/tool_version.py

```diff
@@ -0,0 +1,36 @@
+import sys
+
+import yaml
```

Comment:
hmmm. GHA didn't complain about it. I will need to use this function in my next PR to retrieve version for taskfarmer. If I cannot import, I will update pipfile.

Reply:
Probably because a dependency in the Pipfile has a dependency on it. But that means if we update that dependency and it no longer depends on pyyaml or if we remove that dependency things will break. We should explicitly add it to the Pipfile if we depend on it directly

Reply:
👍 added pyyaml to pipfile.

Reply:
:+1: